lib/lograge/log_subscriber.rb

```diff
@@ -25,13 +19,28 @@ def redirect_to(event)
       Thread.current[:lograge_location] = event.payload[:location]
     end
 
+    def unpermitted_parameters(event)
+      Thread.current[:lograge_unpermitted_params] ||= []
+      Thread.current[:lograge_unpermitted_params].concat(event.payload[:keys])
+    end
+
     def logger
       Lograge.logger.presence || super
     end
 
     private
 
-    def extract_request(payload)
+    def extract_request(event, payload)
```

Comment:
FWIW this could be extracted into an inline class, but I wasn't sure how far it would be worth taking this refactor.